reporters_db/data/reporters.json

```diff
@@ -2094,14 +2095,17 @@
             },
             "examples": [
                 "Bayard's Notebook, 235",
-                "1 Bayard's Notebook, 235"
+                "1 Bayard's Notebook, 235",
+                "Bayard’s Notebook, 235"
```

Comment:
took me a second... this is about the apostrophe?

Reply:
Yes, the json has a unicode quotation mark instead of an apostrophe, the easiest solution was to insert the reporter with unicode character as a solution